README.md

```diff
@@ -2,19 +2,25 @@
 
 # Complete the deputy report
 
-This app is the [Complete the deputy report][service] service. It provides an online reporting service used by deputies to submit their reports, and the private area for case managers to review submitted reports.
+This app is the [Complete the deputy report][service] service. It provides an online reporting service used by deputies
+to submit their reports, and the private area for case managers to review submitted reports.
 
 ## Requirements
 
-You must have Docker installed.
+You must have Docker installed. Once installed click on the cog icon in Docker, select `Experimental Features` and tick
+both boxes. To complete the setup, locate the Digideps root directory in your terminal and run `make up-app-build` or
+the equivalent docker command `docker-compose up -d --build --remove-orphans` which can found in the `Makefile`.
 
-If developing the app then ensure you have [pre-commit](https://pre-commit.com/) installed to take advantage of the pre-commit [hooks](.pre-commit-config.yaml) we've added to the project to make PRs a more consistent and enjoyable experience.
+If developing the app then ensure you have [pre-commit](https://pre-commit.com/) installed to take advantage of the
+pre-commit [hooks](.pre-commit-config.yaml) we've added to the project to make PRs a more consistent and enjoyable
+experience.
```

Comment:
Similar to above - when the markdown is rendered it will wrap as required so hard newlines aren't needed.

```suggestion
If developing the app then ensure you have [pre-commit](https://pre-commit.com/) installed to take advantage of the pre-commit [hooks](.pre-commit-config.yaml) we've added to the project to make PRs a more consistent and enjoyable experience.
```